adserver/models.py

```diff
@@ -393,6 +393,13 @@ class Publisher(TimeStampedModel, IndestructibleModel):
             "Cache this publishers ad requests. Disable for special cases (eg. SaaS users)"
         ),
     )
+    cache_ads_duration = models.PositiveIntegerField(
+        default=0,
```

Comment:
0 seems a bit confusing for a default here, I would expect it to mean no caching or infinite caching. Do we want to use None instead?

Reply:
I try not to have nullable integer fields but if you feel strongly, I'll change it.

Reply:
I don't feel too strongly. Maybe just note it in the docstring.